Jenkinsfile_nightly

```diff
@@ -6,17 +6,25 @@ properties([
 
 @Library("Infrastructure")
 
-def type = "java"
+def type = "nodejs"
```

Comment:
Revert to Java and run functional tests from Gradle.

Reply:
Reverted to Java and added command to run functional tests from Gradle